### DISCUSSION TITLE
Documentation for Adding Custom Animations

### DISCUSSION THREAD

**Discussion by oogami-kero** [Krysiek/CuteDancer#13] [Help] [answered]
If I want to add custom animations for dances I have or have created, is there documentation for the process for adding custom dances? I've seen some others with avatars that have many many dances using CuteDancer, but I'm struggling trying to read the FX and Action Layers added by CuteDancer to add my own.

**Comment by Krysiek** [comment, marked as answer]
Unfortunately there is currently no documentation for this process. It will be available with version 4.0 where adding custom dances is much simpler. In current 4.0-alpha4 version it is possible to generate custom dance template (from Tools -> CuteDancer menu), then you can replace audio and anim files and modify some metadata like name, etc. You can try it, but if you're not an advanced user I recommend waiting for the final release with proper instructions.
